instructor_notes/openscpca/openscpca-setup.md

```diff
@@ -0,0 +1,61 @@
+# Instructor Notes: Set up for OpenScPCA
+
+
+## Learning goals
+
+- Understand how to create and launch a Lightsail for Research (LSfR) instance
+- Be prepared to contribute with an OpenScPCA setup, _ideally_ on LSfR (connectivity permitting)
+- Learn differences between local and LSfR setup
+
+## Activity
+
+For this activity, instructors will demonstrate setting up for OpenScPCA contribution and participants should follow along with all steps.
+At the conclusion of this activity, participants should all have a ready-to-go OpenScPCA setup.
+
+The goal for this activity is to set everyone up on LSfR, but we may need to pivot to local setup depending on the internet connectivity.
+Different potential activity options are below:
+
+### Introduction and LSfR instance setup
+
+1. We will begin by presenting a set of slides outlining all setup steps
+   - Slides will also have explained differences between local setup and LSfR setup
+
+2. Direct participants to [fork](https://openscpca.readthedocs.io/en/latest/technical-setup/fork-the-repo/) `OpenScPCA-analysis`
+
+3. Sign into and familiarize with LSfR:
+    - [Launch a (small) VSCodium LSfR instance](https://openscpca.readthedocs.io/en/latest/software-platforms/aws/creating-vcs/).
+      - Explain that we are using VSCodium since we plan to write a python module, but we could launch an RStudio instance for R development as well.
```

Comment:
```suggestion
      - Explain that we are using VSCodium since we plan to write a Python module, but we could launch an RStudio instance for R development as well.
```